src/pages/language-tour/modules.mdx

```diff
@@ -157,7 +157,7 @@ The content of the Prelude module is documented in [aiken-lang/prelude](https://
 
 The second module that comes with the language is for exposing useful builtin
 functions from Plutus core. Most underlying platform functions are available
-here using a snake case name. Much of Aiken's syntax ends up compiling to
+here using a "snake case" name (eg my_favourite_variable). Much of Aiken's syntax ends up compiling to
```

Comment:
just `"snake_case"` explains it.

Reply:
ok. perhaps keeping quotes

Reply:
I meant here that the name of the casing explains it, not to remove the explanation. E.g: camelCase, snake_case, PascalCase